man/veritytab.xml

```diff
@@ -60,6 +60,13 @@ This is based on crypttab(5).
 
     <variablelist class='fstab-options'>
 
+      <varlistentry>
+        <term><option>hash-offset=<replaceable>BYTES</replaceable></option></term>
+
+        <listitem><para>Offset of hash area/superblock on <literal>hash-device</literal>. Value must be aligned to
+        disk sector offset.</para></listitem>
```

Comment:
what does the latter entence mean? that this should be in multiples of 512? should the last word in the sentence be changed "offset" → "size"?